cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaProduct.js

```diff
@@ -287,16 +287,37 @@ function algoliaProduct(product, fieldListOverride) {
     } else {
         for (var i = 0; i < algoliaFields.length; i += 1) {
             var attributeName = algoliaFields[i];
-            var config = algoliaProductConfig.attributeConfig[attributeName];
+            var config = algoliaProductConfig.attributeConfig[attributeName]
+            var attributeNameObj = attributeName.split('.');
 
-            if (!empty(config)) {
-                var value = null;
-                if (config.localized) {
-                    var currentSites = Site.getCurrent();
-                    var siteLocales = currentSites.getAllowedLocales();
-                    var siteLocalesSize = siteLocales.size();
-                    var currentLocale = request.getLocale();
+            if (empty(config)) {
+                config = jobHelper.getDefaultConfig(attributeName);
```

Comment:
This will throw an error, `getDefaultConfig()` is not defined in `jobHelper`.
Since we don't want to spend time testing v1 jobs, can you revert all changes to this model? So we are sure to not introduce bugs.

Reply:
Removed!